provisioning/atat_provisioning_api.yaml

```diff
@@ -0,0 +1,524 @@
+openapi: 3.0.2
+info:
+  description: >-
+    This is a DRAFT version of the ATAT Portfolio Draft API
+  version: 1.0.0
+  title: ATAT Internal API - Portfolio Drafts
+  contact:
+    email: replaceme@ccpo.mil
+tags:
+  - name: portfolioDraft
+    description: >-
+      Portfolio Drafts represent instances of portfolio provisioning workloads which have not yet been submitted to
+      the CSP
+paths:
+  /portfolioDrafts:
+    get:
+      tags:
+        - portfolioDraft
+      description: Gets all Portfolio Drafts the caller has read access to
+      operationId: getPortfolioDrafts
+      parameters:
+        - $ref: '#/components/parameters/offsetParam'
+        - $ref: '#/components/parameters/limitParam'
+      responses:
+        '200':
+          description: Successful operation with empty response body
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PortfolioDraftSummary'
+        '404':
+          description: Portfolio step not found for this Portfolio Draft or the given Portfolio Draft does not exist
+    post:
+      tags:
+        - portfolioDraft
+      description: Creates a new Portfolio Draft
+      operationId: createPortfolioDraft
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PortfolioDraftSummary'
+      requestBody:
+        description: An empty String or JSON body
+        content:
+          application/json:
+            schema:
+              type: string
+              example: '{}'
+          text/plain:
+            schema:
+              type: string
+              example: ''
```

Comment:
`example: ''`
This is another empty example, could this also be linked to schema?